data/org.worldpossible.Updutil.appdata.xml.in

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>org.worldpossible.Updutil.desktop</id>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>LGPL-2.1-or-later</project_license>
```

Comment:
As a result of containing GPL code copied from Music, the) app is covered by the GPL, so this needs to be changed to match (and the Licence / COPYINNG file